buildscripts/update-data-plane-api-protos.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
This already exists, see [tools/API_SHAS](https://github.com/envoyproxy/java-control-plane/blob/master/tools/API_SHAS) and [tools/update-api.sh](https://github.com/envoyproxy/java-control-plane/blob/master/tools/update-api.sh).